plugins/pipedream/pd.go

```diff
@@ -0,0 +1,25 @@
+package pipedream
+
+import (
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/needsauth"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+)
+
+func PipedreamCLI() schema.Executable {
+	return schema.Executable{
+		Name:    "Pipedream CLI",
+		Runs:    []string{"pd"},
+		DocsURL: sdk.URL("https://pipedream.com/docs/cli/reference/"),
+		NeedsAuth: needsauth.IfAll(
```

Comment:
Let's skip authentication if the command contains "-p" or "--profile" flag. @rajapri28613, are you aware of any other arguments that don't need 1Password authentication?

Reply:
I am not aware of any others.